retrofit-converters/logansquare/src/main/java/retrofit/converter/LoganSquareConverter.java

```diff
@@ -0,0 +1,40 @@
+package retrofit.converter;
+
+import com.bluelinelabs.logansquare.LoganSquare;
+import com.squareup.okhttp.MediaType;
+import com.squareup.okhttp.RequestBody;
+import com.squareup.okhttp.ResponseBody;
+
+import java.io.IOException;
+import java.io.InputStream;
+import java.lang.reflect.Type;
+
+/**
+ * A {@link Converter} which uses LoganSquare for serialization and deserialization of entities.
+ */
+public class LoganSquareConverter implements Converter {
+    private static final MediaType MEDIA_TYPE = MediaType.parse("application/json; charset=UTF-8");
+
+    @Override
+    public Object fromBody(ResponseBody body, Type type) throws IOException {
+        InputStream is = body.byteStream();
+        try {
+            return LoganSquare.parse(is, (Class) type);
```

Comment:
`type` might not always be a `Class`. If the API method returns `List<User>` this will be a `ParamterizedType` which isn't not a `Class`.